src/lib/users/widgets/public_key_selector.rb

```diff
@@ -213,8 +219,10 @@ def blk_devices_combo_box
       #
       # @return [Array<LeafBlkDevice>] List of devices
       def available_blk_devices
-        @available_blk_devices ||= LeafBlkDevice.all.select do |dev|
-          dev.filesystem? && dev.fstype != :squash && dev.transport?
+        if !Yast::Arch.is_wsl
```

Comment:
On WSL this returns `nil`, is that OK? I think this would be safer:

```ruby
return [] if Yast::Arch.is_wsl
```

Reply:
i've changed the code to not need that at all, thanks!